chapter-1-passing-tests.zig

```diff
@@ -460,9 +458,7 @@ test "if optional payload capture" {
     }
 
     const b: ?i32 = 5;
-    if (b) |value| {
-        
-    }
+    if (b) |value| {}
```

Comment:
It's less readable than it was before (just a little), but it's not that significant.